R/Set_up_subfunctions.R

```diff
@@ -0,0 +1,338 @@
+#' @title
+#' Calculate key age parameters to inform life tables (for pig population)
+#' @description
+#' Calculate age rate and age widths for pig compartments depending on number of age groups 
+#'
+#' @param number_age_classes the number of age classes (pigs)
+#' @param pig_age_class_width vector for mean no. of months in each age compartment 
+#'
+#' @return age rate and age width
+#' @export
+age_parameters_pig_func <- function(number_age_classes, pig_age_class_width){
+  
+  number_age_classes_pig = number_age_classes
+  
+  # vector for mean no. of months in each age compartment 
+  
+  age_width_pig <- c()
+  
+  #=======================================#
+  # preparing age_width & age_rate params #
+  
+  # age_width calc: Age-structured model #
+  if (number_age_classes_pig > 1) {
+    for (i in number_age_classes_pig) {
+      # width of 1 month for each age class (symmetrical age classes)
+      age_width_pig[1:number_age_classes_pig] <- pig_age_class_width
+      }
+    }
```

Comment:
I'm not sure what this loop is doing - does it not just replicate `pig_age_class_width`?

Reply:
Agree, loop not required, only `age_width_pig[1:number_age_classes_pig] <- pig_age_class_width`